README.rst

```diff
@@ -72,3 +67,219 @@ response, e.g.
+            },
+            {
+                "Sid": "AllowUserManageBucket",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::[USER ARN]"
+                },
+                "Action": [
+                    "s3:ListBucket",
+                    "s3:GetBucketLocation",
+                    "s3:ListBucketMultipartUploads",
+                    "s3:ListBucketVersions"
+                ],
+                "Resource": "arn:aws:s3:::[BUCKET NAME]"
+            },
+            {
+                "Sid": "AllowUserManageBucketObjects",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::[USER ARN]"
+                },
+                "Action": "s3:*",
+                "Resource": "arn:aws:s3:::[BUCKET NAME]/*"
+            }
+        ]
+    }
+
+
+After the bucket is set up, the bucket can be configured in a Wagtail project.
+
+Set up django-storages
+~~~~~~~~~~~~~~~~~~~~~~
+
+You need to install ``django-storages`` and ``boto3``.
+
+.. code:: sh
+
+   pip install django-storages[boto3]
+
+Set up your S3 bucket with ``django-storages``. The following code will allow
+configuration with environment variables.
+
+.. code:: python
+
+    # settings.py
+    import os
+
+
+    if "AWS_STORAGE_BUCKET_NAME" in os.environ:
+        # Add django-storages to the installed apps
+        INSTALLED_APPS = INSTALLED_APPS + ["storages"]
+
+        # https://docs.djangoproject.com/en/stable/ref/settings/#default-file-storage
+        DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+
+        AWS_STORAGE_BUCKET_NAME = os.environ["AWS_STORAGE_BUCKET_NAME"]
+
+        # Disables signing of the S3 objects' URLs. When set to True it
+        # will append authorization querystring to each URL.
+        AWS_QUERYSTRING_AUTH = False
+
+        # Do not allow overriding files on S3 as per Wagtail docs recommendation:
+        # https://docs.wagtail.io/en/stable/advanced_topics/deploying.html#cloud-storage
+        # Not having this setting may have consequences in losing files.
+        AWS_S3_FILE_OVERWRITE = False
+
+        # Default ACL for new files should be "private" - not accessible to the
+        # public. Images should be made available to public via the bucket policy,
+        # where the documents should use wagtail-storages.
+        AWS_DEFAULT_ACL = "private"
+
+        # We generally use this setting in the production to put the S3 bucket
+        # behind a CDN using a custom domain, e.g. media.llamasavers.com.
+        # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#cloudfront
+        if "AWS_S3_CUSTOM_DOMAIN" in os.environ:
+            AWS_S3_CUSTOM_DOMAIN = os.environ["AWS_S3_CUSTOM_DOMAIN"]
+
+        # When signing URLs is facilitated, the region must be set, because the
+        # global S3 endpoint does not seem to support that. Set this only if
+        # necessary.
```

Comment:
I realise the following is a bit repetitive, but i thought it would be helpful to be more explicit.

```
# When signing URLs is enabled, the region must be set.
# The global S3 endpoint does not seem to support signed URLS.
# Set this only if you will be using signed urls.
```

In the above, is:
 
> is enabled

then accurate/descriptive?